src/pages/DataSamplesPage.tsx

```diff
@@ -3,7 +3,7 @@ import { useCallback, useRef, useState } from "react";
 import { RiAddLine, RiArrowRightLine } from "react-icons/ri";
 import { FormattedMessage } from "react-intl";
 import { useNavigate } from "react-router";
-import DataSampleGridView from "../components/DataSampleGridView";
+import DataSampleGridView from "../components/DataSamplesTable";
```

Comment:
```suggestion
import DataSamplesTable from "../components/DataSamplesTable";
```

And the knock on renaming where the component is used.

Reply:
38d660e6a11da6ab7b0d0a73709e075af3d57fc2